GitUI/Editor/FileViewer.cs

```diff
@@ -982,22 +969,10 @@ private void encodingToolStripComboBox_SelectedIndexChanged(object sender, Event
             Encoding encod = null;
             if (string.IsNullOrEmpty(encodingToolStripComboBox.Text))
                 encod = Module.FilesEncoding;
-            else if (encodingToolStripComboBox.Text.StartsWith("Default", StringComparison.CurrentCultureIgnoreCase))
-                encod = Encoding.Default;
```

Comment:
Please return back `Encoding.Default`, .NET will automatically detect encoding for several cases https://msdn.microsoft.com/en-us/library/system.text.encoding.default.aspx?f=255&mspperror=-2147217396#Anchor_1

`Encoding.Default` should be always in list